package.json

```diff
@@ -37,7 +37,7 @@
   ],
   "scripts": {
     "codecov": "nyc report --reporter=json && codecov -f .coverage/*.json",
-    "docs": "jsdoc -c .jsdoc.js",
+    "docs": "npm run compile && jsdoc -c .jsdoc.js",
```

Comment:
You can also do a `"predocs": "npm run compile"` step to do this :)

Reply:
I learn something every day :)